src/pages/Vaults/VaultsOverview/utils/use-deposit-collateral.tsx

```diff
@@ -0,0 +1,108 @@
+import { CollateralCurrencyExt, CollateralIdLiteral, CurrencyExt, newMonetaryAmount } from '@interlay/interbtc-api';
```

Comment:
Is this hook going to be used for depositing collateral on the dashboard? If so (even if not immediately) this would be better to be located at utils/hooks/api/vaults

Reply:
Not sure yet. What I might reuse is the DepositCollateralStep. So for now I would prefer keeping this here.

Reply:
Ok, fine for now - I'd like to make sure we're locating api hooks consistently, but that will be a case of moving files around so not a big change :) we already have a related ticket in the backlog so we can think about this when we get to it.